client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -84,7 +85,8 @@ const PatternAssembler = ( {
 		() => allPatterns.map( ( pattern ) => encodePatternId( pattern.ID ) ),
 		[ allPatterns ]
 	);
-	const categories = usePatternCategories( site?.ID );
+	const theCategories = usePatternCategories( site?.ID );
+	const categories = useAddCategoryAll( theCategories );
```

Comment:
I guess we can do it inside `usePatternCategories`. Or using the `select` functionality of the react-query to replace the label and description. But it's not a blocker, I'm fine with the current way 🙂

Reply:
I updated the code to use the `select` in the query. I think is better. WDYT?